Update iot-hub-arduino-iot-devkit-az3166-get-started.md by updating d…

jierong opened the pull request:
…ownload link.

The download link for package is not correct, seems https://azureboard.azureedge.net/prod/installpackage/devkit_install_1.1.0.zip should be the correct one.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Waiting for Larry's approval.

rmca14 commented:
Induced merge conflict to sort out changes. I'll meet with Larry to confirm.

rmca14 commented:
Merge conflict resolved in another pull request:
https://github.com/Azure/azure-content/pull/3272#issuecomment-90164186

jierong committed:
Update iot-hub-arduino-iot-devkit-az3166-get-started.md by updating download link.

The download link for package is not correct, seems https://azureboard.azureedge.net/prod/installpackage/devkit_install_1.1.0.zip should be the correct one.
